packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 43.10.0 - August 2, 2018
+
+### Fixed
+
+#### salesforcedx-vscode
+
+- Document how to retrieve code coverage results ([PR #533](https://github.com/forcedotcom/salesforcedx-vscode/pull/533))
+- Update SFDX commands to use ProgressNotification for showing execution progress ([PR #536](https://github.com/forcedotcom/salesforcedx-vscode/pull/536))
```

Comment:
@lcampos - Maybe it would be more useful to say something like this?

`Use ProgressNotification to show progress of SFDX command execution; this change fixes the hanging progress bar that prevented some commands from running successfully ([PR #536](https://github.com/forcedotcom/salesforcedx-vscode/pull/536))`

Reply:
Ah, ok. What effect does this change actually have on customers?

Reply:
Instead of seeing the command execution progress at the bottom of the editor (blue footer) they now get a message box on the bottom right of the editor.

Reply:
Cool, thanks for clarifying. How about something like this?

`Use ProgressNotification to show SFDX command execution progress in a message box instead of in the footer ([PR #536](https://github.com/forcedotcom/salesforcedx-vscode/pull/536))`